src/main/java/at/hannibal2/skyhanni/features/mining/crystalhollows/CrystalHollowsProfessorRobot.kt

```diff
@@ -0,0 +1,52 @@
+package at.hannibal2.skyhanni.features.mining.crystalhollows
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.HypixelCommands
+import at.hannibal2.skyhanni.utils.InventoryUtils
+import at.hannibal2.skyhanni.utils.ItemUtils.name
+import at.hannibal2.skyhanni.utils.LorenzUtils.isInIsland
+import at.hannibal2.skyhanni.utils.StringUtils.removeColor
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class CrystalHollowsProfessorRobot {
+
+    private val config get() = SkyHanniMod.feature.mining.professorRobot;
+
+    private val pattern = "\\[NPC\\] Professor Robot: That's not one of the components I need! Bring me one of the missing components:".toRegex()
```

Comment:
Use a RepoPattern instead. (The key only needs to be unique and somewhat give the direction on what it is)
```suggestion
    private val pattern = RepoPattern.pattern(
    "mining.robot.missing.gfs"
    "\\[NPC\\] Professor Robot: That's not one of the components I need! Bring me one of the missing components:".
    )
```